server/api.go

```diff
@@ -294,6 +294,13 @@ func AddDefaultAccess(a *knox.Access) {
 	defaultAccess = append(defaultAccess, *a)
 }
 
+var accessCallback func(knox.AccessCallbackInput) (bool, error)
+
```

Comment:
I don't love using a global variable for this, but it looks like we don't have a server runtime type struct we pass into all our handlers...so I suppose this is the best we got for now.

Reply:
could be something we think about refactoring in the future